include/deviceconfig.h

```diff
@@ -65,12 +65,12 @@ class DeviceConfig : public IJSONSerializable
     // Add variables for additional settings to this list
     String  hostname = cszHostname;
     String  location = cszLocation;
-    bool    locationIsZip = false;
+    bool    locationIsZip = bLocationIsZip;
     String  countryCode = cszCountryCode;
     String  timeZone = cszTimeZone;
     String  openWeatherApiKey = cszOpenWeatherAPIKey;
-    bool    use24HourClock = false;
-    bool    useCelsius = false;
+    bool    use24HourClock = bUse24HourClock;
+    bool    useCelsius = bUseCelsius;

```

Comment:
In line with my comment about secrets.example.h, I think it's better to leave this as it is now.

Reply:
Same comment as with secrets.example.h